styler.css

```diff
@@ -0,0 +1,217 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family:
+    "Lucida Grande",
+    "Lucida Sans Unicode",
+    Helvetica,
+    Arial,
+    Verdana,
+    sans-serif;
+}
+
+.upper-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 90px 0 150px 0;
+  background: url("images/apple-web-2.png") 0 0 no-repeat;
+  background-size: 100% 100%;
+  background-clip: border-box;
+  background-origin: border-box;
+  margin-bottom: 0;
+}
+
+.container {
+  width: 75%;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+
+.container > nav {
+  width: 100%;
+  display: flex;
+  height: 35px;
+  align-items: center;
+  margin-bottom: 40px;
+  border-radius: 4px;
+  box-shadow: rgba(0, 0, 0, 0.3) 0 1px 2px 0;
+  background:
+    linear-gradient(
+      180deg,
+      rgb(116, 116, 116) 47%,
+      rgb(87, 87, 87) 48%,
+      rgba(137, 137, 137, 1) 120%
+    );
+  font-weight: 600;
+  padding: 0;
+}
+
+footer > div > p > a {
+  color: rgb(0, 136, 204);
+  text-decoration: none;
+}
+
+.help-link-nav > a {
+  flex: 1;
+  display: flex;
+  align-items: center;
+  padding: 5px 40px;
+  font-size: 1vw;
+  color: white;
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
+  border-right: 1px solid gray;
+  height: 100%;
+}
+
+.left-footer-div > p > a {
+  margin: 0 5px;
+}
+
+.right-footer-div > p > a {
+  margin: 0 5px;
+}
+
+footer > div > p > a:hover {
+  text-decoration: underline;
+}
+
+.help-link-nav > a:not(:first-child) {
+  border-left: 1px solid rgba(0, 0, 0, 0.3);
+}
+
+.help-link-nav > a:hover {
+  background-color: rgba(5, 5, 5, 0.4);
+}
+
+.help-link-nav > a:nth-child(1) {
+  font-size: 20px;
+}
+
+footer > div {
+  flex-basis: 42%;
+}
+
+.help-link-nav > div {
+  padding: 5px 10px;
+  border-left: 1px solid rgba(0, 0, 0, 0.3);
+}
+
+.help-link-nav > div > input {
+  width: 80%;
+  background-image: url("images/superior-search.png");
+  background-size: 15px;
+  background-repeat: no-repeat;
+  background-position: 2px 50%;
+  padding: 2px 3px 2px 20px;
+  background-color: transparent;
+  border: 0.5px solid rgb(100, 100, 100);
+  border-radius: 30px;
+  box-shadow: inset 0 2px 1px #333;
+  color: white;
+}
+
+.watch-par > img {
+  margin-left: 10px;
+}
+
+.verse-div > a > img {
+  width: 100%;
+}
+
+.every-div {
+  margin-bottom: 20px;
+}
+
+.every-div > a > img {
+  width: 100%;
+}
+
```

Comment:
We have to clean up the CSS file and remove all the repeated code. Let's follow the DRY principle.